styles/styles.scss

```diff
@@ -104,7 +104,7 @@
     }
 }
 
-body.yc-root {
+body.yc-root:not(.yc-root_mobile):not(.yc-root_native-scrollbar) {
```

Comment:
As a user, I need to create this class name (`.yc-root_native-scrollbar`) manually? How can I find out about this class name and what it does?

Reply:
From documentation. It's just an additional ability to force native scrollbar.
Automatically it's applied only on mobile devices.

Reply:
Where is the documentation? Maybe export this class name?

Reply:
Where I can place this export? Is there an example of such type of class name placement?